CMakeLists.txt

```diff
@@ -16,15 +21,20 @@ set_property(TARGET gpio++-base PROPERTY POSITION_INDEPENDENT_CODE ON)
 # build generic backend
 add_subdirectory(generic)
 
-option(BUILD_PIGPIO "Build pigpio backend" ON)
-if(BUILD_PIGPIO)
-    add_subdirectory(pigpio)
-endif(BUILD_PIGPIO)
+# pigpio should depend on arm architecture only
+if(${CMAKE_SYSTEM_PROCESSOR} MATCHES "arm | aarch64")
+	option(BUILD_PIGPIO "Build pigpio backend" ON)
+	if(BUILD_PIGPIO)
+		add_subdirectory(pigpio)
+	endif()
+endif()
 
 ########################
 # install
-install(DIRECTORY include/ DESTINATION include)
-install(FILES ${FILES} DESTINATION share/doc/${PROJECT_NAME}-${VERSION})
+install(DIRECTORY include/ DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
+install(FILES ${FILES} DESTINATION ${CMAKE_INSTALL_DOCDIR})
+#install(DIRECTORY include/ DESTINATION include)
+#install(FILES ${FILES} DESTINATION share/doc/${PROJECT_NAME}-${VERSION})
```

Comment:
Please don't comment the lines. Just take them out.

Reply:
I totally forgot them